### ISSUE TITLE
Error Analysis on joined final output tables

### ISSUE THREAD

**Issue by lukehsiao** [lukehsiao/HardwareDeepDive#70]
We get [fairly good accuracy results](https://github.com/lukehsiao/HardwareDeepDive/blob/master/doc/README.md#results) on each individual attribute of transistors.

However, the threshold values we use to filter and join up the final table can have very significant impacts on the number of rows we get in the end. For example, changes of +/- 0.1 can result in +/- 100s of rows

I believe that the approach at this point is that we want very high precision in the final tables, then just [run enough data](https://github.com/lukehsiao/HardwareDeepDive/blob/master/doc/README.md#dataset) through that we still end up with a big database of accurate parts.

It would be nice to have a gauge of our final output tables' precision/recall/f-1.

**Comment by lukehsiao** [comment]
In postgres, we can export a query to CSV. For example:

```
\copy (select a.doc, a.part, a.manufacturer, b.polarity, b.ce_v_max, b.cb_v_max, b.eb_v_max, b.c_current_max, b.dev_dissipation, b.stg_temp_min, b.stg_temp_max, b.dc_gain_min from components as a, transistors as b WHERE a.component_id = b.component_id order by a.doc, a.part) TO '~/repos/hwdeepdive/export.csv' CSV HEADER;
```

**Comment by lukehsiao** [comment]
Right now we only get about 13 documents of the 100 into the final table. That's way too few, so I'm going to look into this before looking at the web application.

**Comment by lukehsiao** [comment]
I've improved this to 72 documents. 

Really, no information is lost in joining the tables. It's a matter of how we're defining accuracy at this stage. For example, some rows may missing one or two attributes, and some might have one or two incorrect entries. I'm not sure exactly how we want to measure accuracy.

Some notes for future reference:
- I've set the treshold to `0.7`, which is the same threshold that [we used in our evaluation script](https://github.com/lukehsiao/HardwareDeepDive/blob/143d54a8317d0e975807315161da9ac730222b0e/scripts/evaluate.sh#L50).
- The final tables are created by giving dc_gain_min the highest priority.
  - the results in the dc_gain_min table defines the doc/part_num pairs for which we populate all other values. After these key columns are set, their attributes are just updated from the other tables.
